### PR TITLE
Translate to Azerbaijani: Things I Don’t Know as of 2018

### PR DESCRIPTION
**Pull request by nalekberov**

### CODE REVIEW

**Comment by nalekberov** [comment]
This pull request contains a translation of the article "Things I Don’t Know as of 2018" to Azerbaijani.

File: src/pages/why-isnt-x-a-hook/index.az.md [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+
+However, if `useBailout()` was only respected when *all* calls to it inside a single component “agree” to block an update, our `ChatThread` would fail to update on changes to the `isTyping` prop.
+
+Daha da pisi, bütün bu sematikalarla **`ChatThread`ə `useBailout()` çağırmayan hər hansı bir yeni əlavə edilmiş Hook, onu qırardı.**
+
+**Qərar:** 🔴 `useBailout()` kompozisiyanı qırır. Onu Hooka əlavə etmək digər Hooklarda hal yenilənmələrini qırır. Biz APIların [qırılabilən olmamasını](/optimized-for-change/) istəyirik, bu davranış isə bunun tamamilə əksidir.
+
+### Debaqinq
+
+How does a Hook like `useBailout()` affect debugging?
```
Comment (gaearon) [inline src/pages/why-isnt-x-a-hook/index.az.md]: Can you please extract the unfinished page into a separate PR? Thanks.

Comment (nalekberov) [reply src/pages/why-isnt-x-a-hook/index.az.md]: I just finished it. Sorry for the wait.
